one-pager-maker/src/api/userApi.ts

```diff
@@ -65,7 +65,10 @@ export const userApi = {
         queryKey: [queryKeys.findUserByUID, queryKeys.userUId(uid)],
         queryFn: async () => {
             const result = await userService.findUserByUID(uid);
-            return result.value;
+            if (result && result.value !== undefined) {
+                return result.value;
+            }
+            return null;
```

Comment:
You may not need this change as we discussed at yesterday's meeting

Reply:
I said that yesterday, but the modification was necessary.  
Please allow me to consider the scope of the modifications separately.